cli/run.go

```diff
@@ -134,11 +148,5 @@ func run(c *cli.Context) {
 		printAboutUtilityWorkflows()
```

Comment:
also change the name of `printAboutUtilityWorkflows` to `printAboutUtilityWorkflowsAndExit`